src/app/containers/InlineLink/index.jsx

```diff
@@ -25,8 +25,20 @@ const InlineLinkContainer = ({ locator, isExternal, blocks }) => {
   if (result) {
     // the path is the first item in the array
     const path = result[0];
+    const anchor = locator.split('#')[1];
+    const handleClick = () => {
+      // if there is an anchor,
+      // scroll to the anchored element and focus on it (for the screen readers)
+      if (anchor) {
+        document.getElementById(anchor)?.scrollIntoView();
+        document.getElementById(anchor)?.focus();
```

Comment:
Do we need this scrolling and focus code? Doesn't the browser automatically do this?

Reply:
I would agree this code isn't necessary as I think it is default browser behaviour, I haven't tested this though, I could be wrong?

Reply:
Thanks for the reviews @HarveyPeachey and @andrewscfc. 

Unfortunately neither `react` nor `react-router` do this for us automatically. There is a long standing known issue here about it https://github.com/ReactTraining/react-router/issues/394#issuecomment-220221604. Let's hope they fix it in upcoming updates. 

The other alternative here is using an extension like [react-router-hash-link](https://www.npmjs.com/package/react-router-hash-link) but all that does is `scrollIntoView` and `focus` on the element. It also adds more functionality like timed delays but we won't need that.

Reply:
What would be the reason for not using `react-router-hash-link`? Just would be better to use an external library that can receive updates and track future changes to react router I'd say? Or is your concern an increase in page weight?

If we choose not to use an external library, could link to this in a comment to explain why the code is necessary: https://github.com/ReactTraining/react-router/issues/394#issuecomment-220221604

Reply:
I was worried about the size but it looks like react-router-hash-link would add less than 1.4kb if we use 100% of the library according to Bundlephobia https://bundlephobia.com/result?p=react-router-hash-link@2.4.3 😃 so I will go ahead and use it.